src/Helpers.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Omnipay\PaymentVision;
+
+class Helpers
+{
+    public static function stripDigits(?string $value): ?string
```

Comment:
stripDigits makes it sound like it removes digits.

Reply:
You're right. Let me fix that.

Reply:
https://github.com/RTONational/omnipay-paymentvision/pull/24/commits/6e8c7843021503393fa19180567e3b2e24bd65b2